http_general/test_block_action.py

```diff
@@ -64,18 +66,37 @@ def setup_sniffer() -> analyzer.Sniffer:
         sniffer.start()
         return sniffer
 
+    def check_fin_no_rst_in_sniffer(self, sniffer: analyzer.Sniffer) -> None:
+        sniffer.stop()
+        if not run_config.TCP_SEGMENTATION:
+            self.assert_fin_socks(sniffer.packets)
+            self.assert_unreset_socks(sniffer.packets)
+
+    def check_rst_no_fin_in_sniffer(self, sniffer: analyzer.Sniffer) -> None:
+        sniffer.stop()
+        self.assert_reset_socks(sniffer.packets)
+        self.assert_not_fin_socks(sniffer.packets)
+
 
 class BlockActionReply(BlockActionBase):
+    ERROR_RESPONSE_BODY = ""
+
     def setUp(self):
         self.tempesta = {
             "config": self.tempesta_tmpl % ("reply", "reply"),
         }
         tester.TempestaTest.setUp(self)
 
-    def check_fin_and_rst_in_sniffer(self, sniffer: analyzer.Sniffer) -> None:
-        sniffer.stop()
-        self.assert_fin_socks(sniffer.packets)
-        self.assert_unreset_socks(sniffer.packets)
+    def check_last_error_response(self, client, expected_status_code):
```

Comment:
Here the type hint for method arguments and method return value is not specified. Why? The same can be said for the strings of documentation.

Reply:
I always try to use annotations, but we have not such requirements.